UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -942,6 +970,14 @@ ParseDtb (
     }
   }
 
+  // Post processing: TODO: Need to look into it. Such cross dependency on DT nodes
+  // may not be good idea
+  if (PciRbNode) {

```

Comment:
there may be multiple root bridge nodes so here we may need to record nodes in array and call ParsePciRootBridge multiple times.

Reply:
Ah, good point

Reply:
Done